ideascube/library/static/library/library.css

```diff
@@ -0,0 +1,42 @@
+/* ************************************************* */
```

Comment:
This is exactly what I told @mgautierfr on the parking lot of Pycon, right after the print where he started working on this.

One thing I hadn't paid attention at the time, and that he made me aware of, is that while the monolithic CSS file gets split into 6, each page will at most load 2 CSS files.

My worry about the number of requests went away when I realized that @mgautierfr wasn't adding 5 requests per page but only one.

I don't know if you had realized that or not before making this comment, but since I had fallen for it at first I figured I'd stress it, just to make sure we're all on the same page.